__tests__/frontend-proxy.ts

```diff
@@ -185,355 +90,331 @@ test('uses cookie "frontendUrl" to determine the url of the frontend', async ()
-      const regex = /\/_next\/static\/chunks\/main-[0-9a-f]+.js/
-      const response = await env.fetch({ subdomain: 'en', pathname: '/' })
-      const match = regex.exec(await response.text())
-
-      if (match === null) throw new Error('javascript pathname not found')
-
-      return match[0]
-    }
-  })
-
-  test('/api/frontend/* always resolves to frontend', async () => {
-    const response = await env.fetch({
-      subdomain: 'en',
-      pathname: '/api/frontend/privacy',
-    })
-
-    expect(response.status).toBe(200)
-    expect(await response.json()).toEqual(
-      expect.arrayContaining(['2020-02-10'])
-    )
-    expect(response.headers.get('Set-Cookie')).not.toEqual(
-      expect.stringContaining('useFrontend')
-    )
-  })
-
-  test('/api/auth/* always resolves to frontend (and transfers request header to backend)', async () => {
-    const request = env.createRequest({
-      subdomain: 'en',
-      pathname: '/api/auth/login',
-    })
-    request.headers.set('referer', env.createUrl({ subdomain: 'en' }))
-
+describe('ignore wrongly formatted cookie values', () => {
+  test.each([
+    {
+      cookieValue: 'useLegacyFrontend=foo',
+      backend: Backend.Frontend,
+    },
+    {
+      cookieValue: 'useLegacyFrontend=0.75;',
+      backend: Backend.Frontend,
+    },
+    {
+      cookieValue: 'useLegacyFrontend=1.1;',
+      backend: Backend.Frontend,
+    },
+  ])('Parameters: %p', async ({ cookieValue, backend }) => {
+    const env = currentTestEnvironment()
+    const request = env.createRequest({ subdomain: 'en' })
+    request.headers.set('Cookie', cookieValue)
     const response = await env.fetchRequest(request)
 
-    expect(response.status).toBe(302)
-    expect(response.headers.get('location')).toEqual(
-      expect.stringContaining(env.createUrl({ subdomain: 'hydra' }))
-    )
-    expect(response.headers.get('Set-Cookie')).not.toEqual(
-      expect.stringContaining('useFrontend')
-    )
-  })
-
-  test('/user/notifications always resolve to frontend', async () => {
-    const response = await env.fetch({
-      subdomain: 'en',
-      pathname: '/user/notifications',
-    })
-
-    expect(response.status).toBe(200)
-    expect(await response.text()).toEqual(
-      expect.stringContaining('Notifications')
-    )
+    await expectBackend(response, backend)
+    expect(response.headers.get('Set-Cookie')).toBeNull()
   })
+})
 
-  test('/consent always resolve to frontend', async () => {
+// TODO: Save to remove this?
+// test('chooses legacy backend when type of ressource is unknown', async () => {
+//   givenApi(returnsJson({}))
+//   givenUuid({
+//     __typename: 'Article',
+//     alias: '/unknown',
+//     instance: Instance.En,
+//   })
+
+//   const response = await localTestEnvironment().fetch({
+//     subdomain: 'en',
+//     pathname: '/unknown',
+//   })
+
+//   await expectLegacy(response)
+// })
+
+// TODO: request /en/_assets/favicon.ico but /_assets/favicon.ico is expected currently?
```

Comment:
The tests below fail because of the extra `/en/`.
Not sure what changed to make them fail now.